whylabs_toolkit/monitor/diagnoser/targeting.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Union, Set
+
+from whylabs_toolkit.monitor.models import EntitySchema, ColumnMatrix, DatasetMatrix, TargetLevel
+
+
+def expand_target(target: str, schema: EntitySchema) -> List[str]:
+    if target == "*":
+        return [str(k) for k in schema.columns.keys()]
+    col_items = schema.columns.items()
+    if target == "group:discrete":
+        return [name for (name, c) in col_items if c.discreteness == "discrete"]
+    if target == "group:continuous":
+        return [name for (name, c) in col_items if c.discreteness != "discrete"]
```

Comment:
why `!= discrete` and not `== continuous`?

Reply:
that would indeed be better - fixed